proto/regen/ecocredit/v1/query.proto

```diff
@@ -56,6 +56,14 @@ service Query {
     option (google.api.http).get = "/regen/ecocredit/v1/projects/{project_id}";
   }
 
+  // ProjectsByAdmin queries for all projects by admin with
+  // pagination.
+  rpc ProjectsByAdmin(QueryProjectsByAdminRequest)
+      returns (QueryProjectsByAdminResponse) {
+    option (google.api.http).get =
+        "/regen/ecocredit/v1/projects/by-admin/{admin}";
```

Comment:
I don't have a strong preference but it looks like we have a couple formats for REST endpoints being used. Let's merge this as is and update REST endpoints throughout either using this format or another.

Opened #1152